### PR TITLE
last MOM incubate cards

### PR DESCRIPTION
**Pull request by Northmoc**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/sunder_the_gateway.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+ManaCost:1 W
+Types:Sorcery
+A:SP$ Charm | Choices$ Destroy,Incubate
+
+SVar:Destroy:DB$ Destroy | ValidTgts$ Artifact.OppCtrl+nonToken,Enchantment.OppCtrl+nonToken | TgtPrompt$ Select target nontoken artifact or enchantment an opponent controls | SubAbility$ DBIncubate
+SVar:DBIncubate:DB$ Incubate | Amount$ 2 | StackDescription$ SpellDescription | SpellDescription$ Incubate 2, then transform an Incubator token you control.
+
+SVar:Incubate:DB$ Incubate | Amount$ 2 | StackDescription$ SpellDescription | StackDescription$ SpellDescription | SpellDescription$ Incubate 2, then transform an Incubator token you control.
+SVar:DBTransform:DB$ SetState | Choices$ Incubator.token+YouCtrl | ChoiceTitle$ Choose an Incubator token you control | Mode$ Transform
+
```
Comment (Agetian) [inline forge-gui/res/cardsfolder/upcoming/sunder_the_gateway.txt:11]: Empty lines can be removed

File: forge-gui/res/cardsfolder/upcoming/attentive_skywarden.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Name:Attentive Skywarden
+ManaCost:2 W
+Types:Creature Phyrexian Kor
+PT:2/2
+K:Flying
+T:Mode$ DamageDone | ValidSource$ Card.Self | ValidTarget$ Player,Battle | Execute$ TrigTransform | CombatDamage$ True | TriggerDescription$ Whenever CARDNAME deals combat damage to a player or battle, transform up to one target Incubator token you control.
+SVar:TrigTransform:DB$ SetState | TargetMin$ 0 | TargetMax$ 1 | TgtValidTgts$ Incubator.token+YouCtrl | TgtPrompt$ Select target Incubator token you control | Mode$ Transform
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/attentive_skywarden.txt:7]: TgtValidTgts

File: forge-gui/res/cardsfolder/upcoming/sunder_the_gateway.txt [hunk 3]
```diff
@@ -0,0 +1,13 @@
+Name:Sunder the Gateway
+ManaCost:1 W
+Types:Sorcery
+A:SP$ Charm | Choices$ Destroy,Incubate
+
+SVar:Destroy:DB$ Destroy | ValidTgts$ Artifact.OppCtrl+nonToken,Enchantment.OppCtrl+nonToken | TgtPrompt$ Select target nontoken artifact or enchantment an opponent controls | SubAbility$ DBIncubate
+SVar:DBIncubate:DB$ Incubate | Amount$ 2 | StackDescription$ SpellDescription | SpellDescription$ Incubate 2, then transform an Incubator token you control.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/sunder_the_gateway.txt:7]: Wrong SpellDescription

**Comment by Northmoc** [comment]
Oof… not sure how that version of Sunder the Gateway got through. 🤦 Sorry, fellows

**Review by tool4ever — APPROVED** [review]

**Pull request merged**